src/services/auth.service.js

```diff
@@ -0,0 +1,47 @@
+const jwt = require("jsonwebtoken");
+const bcrypt = require("bcrypt");
+
+const userDao = require("../models/user.dao");
+
+const signUp = async (
+  name,
+  dateOfBirth,
+  phoneNumber,
+  gender,
+  address,
+  email,
+  password
+) => {
+  const user = await userDao.getUserByEmail(email);
+  if (user === undefined) {
```

Comment:
`user === undefined` 는 다음 표현식으로 대체할 수 있습니다!
```suggestion
  if (!user) {
```